packages/twenty-shared/src/constants/TwentyDockerHubLink.ts

```diff
@@ -0,0 +1,2 @@
+export const TWENTY_DOCKER_HUB_LINK =
```

Comment:
I think it doesn't make that much sense to move one but not the other. I would probably have left this one where it was since you answered Paul that it wasn't shared